File: src/main/java/Runner.java
import com.intellij.openapi.actionSystem.AnActionEvent;
import com.intellij.openapi.actionSystem.PlatformDataKeys;
import com.intellij.openapi.editor.Editor;
import com.intellij.openapi.project.Project;
import entry.Builder;

import javax.swing.*;
import java.awt.*;
class Runner {
    void run(AnActionEvent e, Builder builder) {
        Project project = e.getData(PlatformDataKeys.PROJECT);

        Editor editor = e.getData(PlatformDataKeys.EDITOR);
        if (null == editor) {
            return;
        }

        String selectedText = editor.getSelectionModel().getSelectedText();
        String result = builder.gen(selectedText);
        TextCopyForm tf = new TextCopyForm();
        JFrame frame = TextCopyForm.getFrame();

        tf.getT1TextArea().setText(selectedText);
        tf.getT2TextArea().setText(result);

        frame.setContentPane(tf.getPanel1());
        frame.setPreferredSize(new Dimension(1100, 600));
        frame.setExtendedState(JFrame.NORMAL);

        frame.pack();
        frame.setLocationRelativeTo(null);
        frame.setVisible(true);
    }
}

File: src/main/kotlin/Runner.kt
import com.intellij.openapi.actionSystem.AnActionEvent
import com.intellij.openapi.actionSystem.PlatformDataKeys
import entry.Builder

internal class Runner {
    fun run(e: AnActionEvent, builder: Builder) {
        val project =
            e.getData(PlatformDataKeys.PROJECT)
        val editor = e.getData(PlatformDataKeys.EDITOR) ?: return
        val selectedText = editor.selectionModel.selectedText
        val tf = TextCopyForm(project)
//        val framee = TextCopyForm.getFrame()
        tf.setBuilder(builder,selectedText)
//        val p = Objects.requireNonNull(WindowManager.getInstance().getFrame(project))
//            ?.locationOnScreen
//        frame.location = project
//        tf.t1TextArea.text = selectedText
        tf.gen()

        tf.pack()
        tf.show()
      //  tf.setBuilder(builder)

       // frame.contentPane = tf.panel1
        //frame.preferredSize = Dimension(1100, 600)
        //frame.extendedState = JFrame.NORMAL
        //frame.pack()
        //frame.isVisible = true
    }
}